src/main/java/androidx/view/ViewGroup.kt

```diff
@@ -114,3 +116,12 @@ fun ViewGroup.MarginLayoutParams.updateMarginsRelative(
     marginEnd = end
     bottomMargin = bottom
 }
+
+/**
+ * Inflates the given layoutId into the ViewGroup.
+ *
+ * @see LayoutInflater.inflate
+ */
+fun ViewGroup.inflate(@LayoutRes layoutId: Int, attachToRoot: Boolean = true): View {
```

Comment:
I like attachToRoot being true by default. I would however rename it here to attachToSelf or attachToThis instead? (or any other better idea you might have)